From: Stephane Glondu <steph@glondu.net>
Date: Tue, 19 Apr 2011 12:45:34 +0200
Subject: Fix thumb2-related build error

Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=622882
Signed-off-by: Konstantinos Margaritis <markos@genesi-usa.com>
Signed-off-by: Stephane Glondu <steph@glondu.net>
---
 kernel/byterun/coq_interp.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/kernel/byterun/coq_interp.c b/kernel/byterun/coq_interp.c
index cce2319..a0cb4f1 100644
--- a/kernel/byterun/coq_interp.c
+++ b/kernel/byterun/coq_interp.c
@@ -145,7 +145,7 @@ sp is a local copy of the global variable extern_sp. */
 #define SP_REG asm("a4")
 #define ACCU_REG asm("d7")
 #endif
-#ifdef __arm__
+#if defined(__arm__) && !defined(__thumb2__)
 #define PC_REG asm("r9")
 #define SP_REG asm("r8")
 #define ACCU_REG asm("r7")
-- 
